rebar.config.script

```diff
@@ -1,8 +0,0 @@
-Artifacts =  case os:type() of
-    {win32, _} ->
-        [{artifacts, ["priv/fast_pbkdf2.dll"]}];
-    _ ->
-        [{artifacts, ["priv/fast_pbkdf2.so"]}]
-end,
-
-CONFIG ++ Artifacts.
```

Comment:
@big-r81 can you check that without this it still works for rebar2? I checked source-code for rebar2 and for rebar3+pc and the filetype should be replaced automatically, see for example https://github.com/blt/port_compiler/blob/master/src/pc_port_specs.erl#L220

Reply:
@NelsonVides I will test this. The idea behind the "artifacts" isn't the correct naming of the dynamic library (this works with and without the script). It's for the "status" of the build, if the binaries are build correctly. If the build fails, it will give a warning "Hey, the binaries aren't produced" ... And I can't use e.g. an "*" as "priv/fast_pbkdf2.*" as static artifacts "command" in rebar.config.

Reply:
All right, will push this PR without the commit removing this, doesn't bother much and we can always remove it later if it turns out to be unneeded. Will merge this when CI passes, thanks for your help 😄